frontend/model/contracts/group.js

```diff
@@ -259,6 +261,29 @@ DefineContract({
         }
       }
     },
+    'gi.contracts/group/updateSettings': {
+      // OPTIMIZE: Make this custom validation function
+      // reusable accross other future validators
```

Comment:
It's not clear to me what this comment is suggesting, as these validations are specific only to this action, and shouldn't be reused elsewhere (AFAIK)

Reply:
I don't mean reusing the validator itself, but the way we create a custom validator. It's quite verbose at the moment. I'll think about something some declarative on a future task ;)